src/config/dynamic-theme-fixes.config

```diff
@@ -2819,6 +2819,17 @@ INVERT
 
 ================================
 
+chessprogramming.org
+
+INVERT
+img:not(.thumbimage):not(.thumbborder)
```

Comment:
This will invert all kind of images, and not just the ones with latex expressions on it. This is too broad to be included.

Reply:
I tried to narrow the selector down as far as possible in a new commit. As far as I looked through the pages all images that aren't equations or diagrams integrated into the text have the `thumbborder` or `thumbimage` class set.